core/extensions.js

```diff
@@ -103,8 +103,10 @@ Blockly.Extensions.registerMutator = function(name, mixinObj, opt_helperFn,
   var errorPrefix = 'Error when registering mutator "' + name + '": ';
 
   // Sanity check the mixin object before registering it.
-  Blockly.Extensions.checkHasFunction_(errorPrefix, mixinObj, 'domToMutation');
-  Blockly.Extensions.checkHasFunction_(errorPrefix, mixinObj, 'mutationToDom');
```

Comment:
When compiled, "foo.domToMutation" becomes "f.g".  Passing a property name to this function as a string will not work.